charts/ratify/templates/deployment.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   name: {{ include "ratify.fullname" . }}
   labels:
     {{- include "ratify.labels" . | nindent 4 }}
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
```

Comment:
why is the deployment a helm hook? isn't the deployment the main resource to be installed after all other dependencies?

Reply:
you're right, just found out my previous error was due to a failure to delete ratify deployment, which caused my new deployment fail without this annotation.